docs/source/coroutines.rst

```diff
@@ -194,7 +194,7 @@ The second section in the above code using it would be ``await with_timeout(quie
 Determining Which Task Finishes First
 -------------------------------------
 
-:class:`~cocotb.triggers.First` can be used to determine which of multiple Tasks :meth:`~cocotb.task.Task.complete` first using the following idiom.
+:class:`~cocotb.triggers.First` can be used to determine which of multiple Tasks :meth:`complete <~cocotb.task.Task.complete>` first using the following idiom.
```

Comment:
Strange this doesn't work like classes or methods?
```suggestion
:class:`~cocotb.triggers.First` can be used to determine which of multiple Tasks :meth:`complete <cocotb.task.Task.complete>` first using the following idiom.
```

Reply:
If you mean that `~` means to strip all elements from the left except the last - it does that, but it ends up with `complete()`, and this change removes the `()`.